receiver/sqlserverreceiver/queries.go

```diff
@@ -210,6 +212,36 @@ SELECT DISTINCT
 
 INSERT INTO @PCounters SELECT * FROM PerfCounters;
 
+SET  @SqlStatement = N'
+SELECT
+	 ''SQLServer:Workload Group Stats'' AS [object]
+	,[counter]
+	,[instance]
+	,CAST(vs.[value] AS BIGINT) AS [value]
+	,1
+FROM
+(
+    SELECT
+		 rgwg.name AS instance
+		,rgwg.total_request_count AS [Request Count]
```

Comment:
Including extra selects here in case we want to add these metrics in the future.